moto/s3/models.py

```diff
@@ -2315,11 +2320,7 @@ def put_bucket_notification_configuration(
          - AWSLambda
          - SNS
          - SQS
-
```

Comment:
We should keep the documentation that only `s3:ObjectCreated:Copy` and `s3:ObjectCreated:Put` are supported. 

Those are still the only two methods where we actually invoke `s3/notifications.py::send_event`.

Reply:
@bblommers 
That's true, thanks